pkg/controller/controller_utils.go

```diff
@@ -1047,3 +1048,53 @@ func ComputeHash(template *v1.PodTemplateSpec, collisionCount *int32) uint32 {
 
 	return podTemplateSpecHasher.Sum32()
 }
+
+func MergeResourceChanges(pod *v1.Pod, cMap map[string]*v1.Container) []v1.Container {
```

Comment:
Please rename function to GetUpdatedPodResources

Reply:
fixed